src/tile.js

```diff
@@ -1,14 +1,7 @@
 import React from 'react';
-import './tile.css';
-import RecipeWindow from './Components/MealDB/recipewindow.js'
-
-function CheckError(response) {
-  if (response.status >= 200 && response.status <= 299) {
-    return response.json();
-  } else {
-    throw Error(response.statusText);
-  }
-}
+import styles from './tile.css';
```

Comment:
styles isn't defined so maybe just import './tiles.css' similar to recipeWindow.css?

Reply:
also if you could removed flipped from Tile's state since it's not being used